src/Components/v2/__stories__/CarouselV2.story.tsx

```diff
@@ -0,0 +1,186 @@
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+          />
+        </Box>
+      </Section>
+      <Section title="Artwork Brick">
+        <Box width="70%">
+          <RelayStubProvider>
+            <Carousel
+              data={artworkBricks}
+              render={artwork => {
+                const {
+                  node: {
+                    image: { aspect_ratio },
+                  },
+                } = artwork
+
+                return (
+                  <FillwidthItem
+                    artwork={artwork.node}
+                    targetHeight={200}
+                    imageHeight={200}
+                    width={200 * aspect_ratio}
+                    margin={20}
+                  />
+                )
+              }}
+            />
+          </RelayStubProvider>
+        </Box>
+      </Section>
+      <Section title="Custom Arrows with defaults">
+        <Box width="70%">
+          <Carousel
+            data={images}
+            render={props => {
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+            renderLeftArrow={({ Arrow }) => {
+              return (
+                <Box top={10} position="relative" bg="black10">
+                  <Arrow />
+                </Box>
+              )
+            }}
+            renderRightArrow={({ Arrow }) => {
+              return (
+                <Box top={10} position="relative" bg="black10">
+                  <Arrow />
+                </Box>
+              )
+            }}
+          />
+        </Box>
+      </Section>
+      <Section title="Custom Arrows ">
+        <Box width="70%">
+          <Carousel
+            data={images}
+            render={props => {
+              return (
+                <Image
+                  px={5}
+                  src={props.resized.url}
+                  width={props.resized.width}
+                  height={props.resized.height}
+                />
+              )
+            }}
+            renderLeftArrow={({ getFlickity }) => {
+              return (
+                <Box
+                  onClick={() => {
+                    getFlickity && getFlickity.previous()
+                  }}
+                >
+                  Prev
+                </Box>
+              )
+            }}
+            renderRightArrow={({ getFlickity }) => {
+              return (
+                <Box
+                  onClick={() => {
+                    getFlickity && getFlickity.next()
+                  }}
+                >
+                  Next
+                </Box>
+              )
+            }}
+          />
+        </Box>
+      </Section>
```

Comment:
If it's easy, can we add another entry here that _only_ renders `CarouselServer`?